ccm_web/client/src/api.ts

```diff
@@ -30,3 +34,19 @@ export const getGlobals = async (key: string | undefined): Promise<Globals> => {
   await handleErrors(resp)
   return await resp.json()
 }
+
+const delay = async (ms: number): Promise<void> => {
+  await new Promise<void>(resolve => setTimeout(() => resolve(), ms))
+}
+
+// This is a placeholder for a real implementation (I mean, obviously :D)
+export const getCourseSections = async (key: string | undefined, courseId: string): Promise<string[]> => {
+  const sections = await delay(2000).then(() => {
+    if (Math.random() * 2 > 1) {
+      return (['AAAA', 'BBBB'])
+    } else {
+      return new Promise<string[]>((resolve, reject) => { reject(new Error('Error retrieving course section information.')) })
+    }
+  })
+  return sections
```

Comment:
It could certainly change, the only thing I think the bulk section create feature needs is the section names, so that is what I had it returning.  If we want to reuse this for something else it could definitely make sense to incorporate more info in the response.